hardware/P1MeterBase.cpp

```diff
@@ -13,15 +15,15 @@ typedef enum {
 
 #define P1_SMID			"/" // Smart Meter ID. Used to detect start of telegram.
 #define P1PU1			"1-0:1.8.1" // total power usage normal tariff
-#define P1PU2			"1-0:1.8.2" // total power usage low tariff
+#define P1PU2			"1-0:1.8.2" // total power usage day/high tariff
```

Comment:
It is not day/high tariff, but actually normal/low or low/high (see the weekend)

The documentation is not very clear about this, its called tariff 1 / tariff 2, later normal/high ... ohh and also low and normal/high

http://files.domoticaforum.eu/uploads/Smartmetering/DSMR%20v4.0%20final%20P1.pdf

maybe we should follow chapter 6.1 Electricity data ?
or the labeling in paragraph 6

6 DATA OBJECTS
Currently two tariffs (Rate 1 low tariff and Rate 2 normal/high tariff)
are defined

Or Rate 1 and Rate 2 ?